src/v/cluster/topic_table.cc

```diff
@@ -66,8 +66,8 @@ topic_table::apply(create_topic_cmd cmd, model::offset offset) {
     for (auto& pas : md.get_assignments()) {
         auto ntp = model::ntp(cmd.key.ns, cmd.key.tp, pas.id);
         replicas_revision_map replica_revisions;
+        _partition_count++;
         for (auto& r : pas.replicas) {
-            _partition_count++;
```

Comment:
>cluster: increment redpanda_cluster_partitions per partition on topic creation

@daisukebe would be great in the future to add a sentence or two of context to commit messages that explain what was wrong and why the change fixes it.

Reply:
Indeed, will do. Thanks for the heads up.